x-pack/test/detection_engine_api_integration/utils.ts

```diff
@@ -7,8 +7,8 @@
 import { Client } from '@elastic/elasticsearch';
 import { SuperTest } from 'supertest';
 import supertestAsPromised from 'supertest-as-promised';
-import { OutputRuleAlertRest } from '../../../../plugins/siem/server/lib/detection_engine/types';
-import { DETECTION_ENGINE_INDEX_URL } from '../../../../plugins/siem/common/constants';
+import { OutputRuleAlertRest } from '../../plugins/siem/server/lib/detection_engine/types';
+import { DETECTION_ENGINE_INDEX_URL } from '../../plugins/siem/common/constants';
```

Comment:
NOTE: I pulled up utils two directories so that each test scenario such as "basic", "security-and-spaces", anything else will be able to use the same utils package for rules.